package/src/components/MessageInput/__tests__/UploadProgressIndicator.test.js

```diff
@@ -57,24 +57,48 @@ describe('UploadProgressIndicator', () => {
     });
   });
 
-  it('should render an active IN_PROGRESS UploadProgressIndicator', async () => {
+  it('should render an active UploadProgressIndicator and not-supported indicator', async () => {
     const action = jest.fn();
 
     const { queryByTestId, toJSON } = render(
       <ThemeProvider>
         <UploadProgressIndicator
           action={action}
-          active
-          type={ProgressIndicatorTypes.IN_PROGRESS}
+          type={ProgressIndicatorTypes.NOT_SUPPORTED}
         ></UploadProgressIndicator>
       </ThemeProvider>,
     );
 
     await waitFor(() => {
       expect(queryByTestId('active-upload-progress-indicator')).toBeTruthy();
+      expect(queryByTestId('not-supported-upload-progress-indicator')).toBeTruthy();
```

Comment:
I think it would be possible to query for the `File is not supported` text (or what the error message says), right? If so, we don't need the testID :)

Reply:
✅

Reply:
I don't think we can `queryByText` here because no text is being displayed in this component as it is.

Reply:
I also checked out running the tests using accessibility label and state and that didn't suffice so I might leave these tests as they are. I am also open to any suggestions on how to make these tests better.

After going through it for a bit longer I figured I could add more tests to the FileUploadPreview and ImageUploadPreview components to better fit with what have now.